src/netex_idf/stops.rs

```diff
@@ -108,6 +237,53 @@ pub fn from_path(path: &std::path::Path, collections: &mut Collections) -> Resul
 mod tests {
     use super::*;
 
+    #[test]
+    #[should_panic(expected = "longitude and latitude not found")]
+    fn test_load_coords_with_one_coord() {
+        let xml = r#"
+<Quay>
+    <Centroid>
+        <Location>
+            <gml:pos srsName="EPSG:2154">666944.0</gml:pos>
+        </Location>
+    </Centroid>
+</Quay>"#;
+        let root: Element = xml.parse().unwrap();
+        load_coords(&root).unwrap();
+    }
+
+    #[test]
+    #[should_panic]
+    fn test_load_unvalid_coords() {
+        let xml = r#"
+<Quay>
+    <Centroid>
+        <Location>
+            <gml:pos srsName="EPSG:2154">666944.0 ABC</gml:pos>
+        </Location>
+    </Centroid>
+</Quay>"#;
+        let root: Element = xml.parse().unwrap();
+        load_coords(&root).unwrap();
+    }
+
+    #[test]
+    #[should_panic]
+    fn test_load_coords() {
```

Comment:
I don't understand why this test should panic. Could you find a more explicit name for the function?

Reply:
nothing to understand, just a bad copy paste